backend/django_core/static_src/components/page/explore/Index.svelte

```diff
@@ -0,0 +1,220 @@
+<script lang="ts">
+    import { cn } from "$functions/classname";
+    import ScrollArea from "$components/minor/ScrollArea/Index.svelte";
+    import Tick from "$icons/Tick/Index.svelte";
+    import MoreBox from "$icons/MoreBox/Index.svelte";
+
+    // Mapping
+    let filter_options_mapping: {
+        [key: string]: {
+            title: string;
+            class: string;
+            value: string;
+            items?: Record<string, string> | undefined;
```

Comment:
items are not `undefined`, it should be `null`

Reply:
This `items` is already an optional type, (`undefined`), huh?

Reply:
It should still be null by default